### PR TITLE
correcting translation issues

### PR DESCRIPTION
**Pull request by dsnyder-fairway**

### CODE REVIEW

File: webapp/src/main/webapp/src/app/shared/layout/sb-breadcrumbs.component.ts [hunk 1]
```diff
@@ -78,6 +78,9 @@ export class SbBreadcrumbs implements OnInit {
     ).subscribe(() => {
       this.breadcrumbs = this.createBreadcrumbs(this.activatedRoute.root);
     });
+    this.translateService.onLangChange.subscribe( () => {
```
Comment (dsnyder-fairway) [inline webapp/src/main/webapp/src/app/shared/layout/sb-breadcrumbs.component.ts]: translates breadcrumbs on language toggle

File: webapp/src/main/webapp/src/app/student/results/student-results.component.html [hunk 2]
```diff
@@ -53,7 +53,7 @@ <h3>{{'student-results.title' | translate}}</h3>
       <div class="well wide-content-container">
         <span class="h3 label-group"
               [ngClass]="getAssessmentTypeColor(assessmentType)">
-          <span class="label">{{'common.assessment-type.' + assessmentType + '.short-name' | translate}}</span><span class="label">{{subject}}</span>
+          <span class="label">{{'common.assessment-type.' + assessmentType + '.short-name' | translate}}</span><span class="label">{{'common.subject.' + subject + '.short-name' | translate}}</span>
```
Comment (dsnyder-fairway) [inline webapp/src/main/webapp/src/app/student/results/student-results.component.html]: note -- this changes MATH to Math. I am not sure if this is a concern. I can change this if it is

**Comment by dsnyder-fairway** [comment]
other issues are within the es.json file

File: webapp/src/main/webapp/src/app/student/results/student-results.component.html (hunk 2 above)
Comment (esotrope) [reply webapp/src/main/webapp/src/app/student/results/student-results.component.html]: 👍 as it should : j

**Pull request merged**